tests/UI/pages/BO/orders/add.js

```diff
@@ -131,6 +158,56 @@ class AddOrder extends BOBasePage {
       this.waitForHiddenSelector(page, this.customerCardChooseButton(cardPosition)),
       this.waitForVisibleSelector(page, this.checkoutHistoryBlock),
     ]);
+
+    return this.elementVisible(page, this.checkoutHistoryBlock, 1000);
+  }
+
+  /**
+   * Click on details button
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  async clickOnDetailsButton(page) {
+    await this.waitForSelectorAndClick(page, this.customerCardDetailButton);
+
+    return this.elementVisible(page, this.customerIframe, 2000);
+  }
+
+  /**
+   * Get personal information title
+   * @param page {Page} Browser tab
+   * @param customerID {number} Id of customer to check details
+   * @returns {Promise<string>}
+   */
+  async getPersonalInformationTitle(page, customerID) {
+    const viewCustomerFrame = await page.frame({url: new RegExp(`sell/customers/${customerID}/view`, 'gmi')});
+
+    return viewCustomerPage.getPersonalInformationTitle(viewCustomerFrame);
+  }
```

Comment:
Why not create a function to get the iframe, that call it directly on the test and use `viewCustomerPage` also on the test ?

Reply:
Thanks! good idea I will change it